manifests/params.pp

```diff
@@ -0,0 +1,22 @@
+#
+#
+# == Class: artifactory::params
+#
+# The params class of the artifactory module.
+#
+#
+class artifactory::params (
+    $artifactory_base       = '/var/opt/jfrog/artifactory',
```

Comment:
meaning the container always (!) has the same directories, so being able to change them does not make sense. static layout, in a way. anyway, I think this is a rare case anyway...